specification/synapse/data-plane/Microsoft.Synapse/preview/2019-11-01-preview/roleAssignments.json

```diff
@@ -0,0 +1,354 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-11-01-preview",
+    "title": "SynapseClient"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{workspaceName}.{SynapseDnsSuffix}",
+    "parameters": [
+      {
+        "name": "workspaceName",
+        "description": "The name of the workspace to execute operations on.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      },
+      {
+        "$ref": "#/parameters/SynapseDnsSuffixInPath"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/rbac/roleAssignments": {
+      "post": {
+        "tags": [
+          "RoleAssignments"
+        ],
+        "operationId": "RoleAssignments_CreateRoleAssignment",
+        "description": "Create role assignment.",
+        "x-ms-examples": {
+          "Create role assignment": {
+            "$ref": "./examples/RoleAssignments_CreateRoleAssignment.json"
+          }
+        },
+        "consumes": [
+          "application/json",
+          "text/json"
```

Comment:
I would remove "text/json", that is not a recommended mimetype and was in old Swagger for compat reason, I can't suggest it on new Swagger

Reply:
> I would remove "text/json", that is not a recommended mimetype and was in old Swagger for compat reason, I can't suggest it on new Swagger

Removed